.github/workflows/upload-to-secure-pipelines.yml

```
@@ -23,7 +23,7 @@ jobs:
       - name: Upload Cognito SAM package
         uses: alphagov/di-github-actions/secure-pipelines/upload-sam-package@78978571966007f342dd46b61c8c4d2d5e5da81b
         with:
-          template: backend/cognito/cognito.template.yml
+          template: backend/cognito/template.yml
```

```suggestion
          template: backend/cognito/cognito.template.yml
```
This can be reverted for this PR, out of scope

---

addressed